src/classlibnative/bcltype/system.cpp

```diff
@@ -41,14 +41,35 @@ typedef BOOL (*pfnGetPhoneVersion)(LPOSVERSIONINFO lpVersionInformation);
 pfnGetPhoneVersion g_pfnGetPhoneVersion = NULL;
 #endif
 
+typedef void(WINAPI *pfnGetSystemTimeAsFileTime)(LPFILETIME lpSystemTimeAsFileTime);
+pfnGetSystemTimeAsFileTime g_pfnGetSystemTimeAsFileTime = NULL;
+Volatile<bool> g_fGetSystemTimeAsFileTimeInitialized = false;
 
 FCIMPL0(INT64, SystemNative::__GetSystemTimeAsFileTime)
```

Comment:
Meaning change the API signature up through to how its called from managed?  Makes sense, but I'd like to leave that for a separate change, and leave this one just as the precision change.